fitlins/viz/reports.py

```diff
@@ -142,9 +142,11 @@ def write_full_report(report_dict, run_context, deriv_dir):
 
     tpl = env.get_template('data/full_report.tpl')
 
-    model = snake_to_camel(report_dict['model']['name'])
+    model_name = snake_to_camel(report_dict['model']['name'])
+    if model_name == "":
+        model_name = "untitled"
```

Comment:
We can even tolerate a missing `'name'` altogether:

```suggestion
    model_name = snake_to_camel(report_dict['model'].get('name') or "untitled")
```